encoding/codecs/x264.md

```diff
@@ -0,0 +1,292 @@
+# Encoding with x264
+
+H.264 has been the de facto standard video format
+across the internet for the past decade.
+It is widely supported for playback in all modern browsers
+and many hardware devices such as gaming consoles and phones.
+It provides better video quality at smaller file sizes
+compared to its predecessors.
+
+x264 is a mature, free, open-source encoder
+for the H.264 video format.
+This guide will focus on x264.
```

Comment:
I think this line is redundant, or should at least be changed to say `This page`

Reply:
:+1:, also in at least one other location.